e2e/js/src/js-ts-solution.test.ts

```diff
@@ -129,23 +129,6 @@ ${content}`
       `Successfully ran target build for project @proj/${viteParentLib} and 5 tasks it depends on`
     );
 
-    // check typecheck
-    expect(runCLI(`typecheck ${esbuildParentLib}`)).toContain(
```

Comment:
Why are we removing all typecheck tasks? Should we only remove the one from `tscParentLib`?